.gitignore

```diff
@@ -6,7 +6,7 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 .pnpm-debug.log*
-# package-lock.json
+package-lock.json

```

Comment:
Litt usikker på om det funker uten denne i CI pipeline. Men det får du bare teste

Reply:
Lockfiler bør være i git loggen
https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for

Reply:
hmmmm, ok da, selv om jeg ikke er så fan av mentaliteten at vi skal ha en evig voksende fil som installerer eldre versjoner fordi det får koden til å kjøre. En burde jo ha tilnærmingen at vi fikser dependency-problemer fortløpende, men er nok bedre at alle får den til å kjøre på dugnad i stedet for å måtte fikse den hver dugnad.